From: Aristeu Rozanski <aris@redhat.com>
Date: Wed, 18 Nov 2009 22:02:38 -0500
Subject: [Fedora] [input] remove unwanted messages on spurious events
Message-id:
Patchwork-id:
O-Subject:
Bugzilla:

Original Fedora patch: linux-2.6-input-kill-stupid-messages.patch

Quite a few key switchers and other tools trigger this and it confuses
people who can do nothing about it.

Signed-off-by: Aristeu Rozanski <arozansk@redhat.com>

diff --git a/drivers/input/keyboard/atkbd.c b/drivers/input/keyboard/atkbd.c
index 28e6110..cbe3989 100644
--- a/drivers/input/keyboard/atkbd.c
+++ b/drivers/input/keyboard/atkbd.c
@@ -424,10 +424,14 @@ static irqreturn_t atkbd_interrupt(struct serio *serio, unsigned char data,
 			goto out;
 		case ATKBD_RET_ACK:
 		case ATKBD_RET_NAK:
+#if 0
+			/* Quite a few key switchers and other tools trigger this and it confuses
+			   people who can do nothing about it */
 			if (printk_ratelimit())
 				printk(KERN_WARNING "atkbd.c: Spurious %s on %s. "
 				       "Some program might be trying access hardware directly.\n",
 				       data == ATKBD_RET_ACK ? "ACK" : "NAK", serio->phys);
+#endif
 			goto out;
 		case ATKBD_RET_ERR:
 			atkbd->err_count++;
